projects/steering/test/test_steering_can.c

```diff
@@ -0,0 +1,63 @@
+#include "can.h"
+#include "can_msg_defs.h"
+#include "event_queue.h"
+#include "exported_enums.h"
+#include "gpio.h"
+#include "gpio_it.h"
+#include "interrupt_def.h"
+#include "ms_test_helpers.h"
+#include "soft_timer.h"
+#include "status.h"
+#include "steering_can.h"
+#include "steering_digital_input.h"
+#include "steering_events.h"
+#include "test_helpers.h"
+
+#define REGEN_BRAKING_TOGGLE_CAN \
+  { .id = 12, .data = 0 }
+
+static CanStorage s_can_storage;
+
+static CanSettings can_settings = { .device_id = SYSTEM_CAN_DEVICE_STEERING,
+                                    .bitrate = CAN_HW_BITRATE_500KBPS,
+                                    .rx_event = STEERING_CAN_EVENT_RX,
+                                    .tx_event = STEERING_CAN_EVENT_TX,
+                                    .fault_event = STEERING_CAN_FAULT,
+                                    .tx = { GPIO_PORT_A, 12 },
+                                    .rx = { GPIO_PORT_A, 11 },
+                                    .loopback = true };
+
+static int count = 0;
```

Comment:
I like that you're using a static count to make sure the regen braking toggle request was sent! A couple points about it:
* you should reset it to 0 in setup_test in case other tests use it in the future
* please use a fixed-width int type like `uint8_t` instead of `int` so it's consistent on x86 and stm32
* since it's a static variable, it should be prefixed with `s_`. Also, `count` is kind of a generic name - how about something that gives an indication of its use a bit more, like `s_msg_rx_count`?